docs/source/man.rst

```diff
@@ -387,25 +384,32 @@ The following options are available for every Quipucords command.
 Examples
 --------
 
-:Creating a new network type credential with a keyfile: ``qpc cred add --name=new-creds **--type=** *network* --username=qpc-user --sshkeyfile=/etc/ssh/ssh_host_rsa_key``
-:Creating a new network type credential with a password: ``qpc cred add --name=other-creds **--type=** *network* --username=qpc-user-pass --password``
-:Creating a new vcenter type credential: ``qpc cred add --name=vcenter-cred **--type=** *vcenter* --username=vc-user-pass --password``
-:Creating a new network source: ``qpc source add --name=new-source --type network --hosts 1.192.0.19 1.192.0.20 --cred new-creds``
-:Creating a new vcenter source: ``qpc source add --name=new-source --type vcenter --hosts 1.192.0.19 --cred vcenter-cred``
-:Editing a source: ``qpc source edit --name=new-source --hosts 1.192.0.[0:255] --cred new-creds other-creds``
-:Running a scan with a source: ``qpc scan start --sources new-source``
+Creating a new network type credential with a keyfile
+  ``qpc cred add --name=new_creds --type=network --username=qpc_user --sshkeyfile=/etc/ssh/ssh_host_rsa_key``
+Creating a new network type credential with a password
+  ``qpc cred add --name=other_creds --type=network --username=qpc_user_pass --password``
+Creating a new vcenter type credential
+  ``qpc cred add --name=vcenter_cred --type=vcenter --username=vc-user_pass --password``
+Creating a new network source
+  ``qpc source add --name=new_source --type network --hosts 1.192.0.19 1.192.0.20 --cred new_creds``
+Creating a new vcenter source
+  ``qpc source add --name=new_source --type vcenter --hosts 1.192.0.19 --cred vcenter_cred``
+Editing a source
+  ``qpc source edit --name=new_source --hosts 1.192.0.[0:255] --cred new_creds other_creds``
+Running a scan with one source
+  ``qpc scan start --sources new_source``
 
 Security Considerations
 -----------------------
 
-The credential credentials that are used to access servers are stored with the source in an AES-256 encrypted value within a database. A vault password is used to encrpyt/decrypt values. The vault password and decrypted values are in the system memory, and could theoretically be written to disk if memory swapping is enabled.
+The authentication data in the credentials and the network-specific and system-specific data in sources are stored in an AES-256 encrypted value within a database. A vault password is used to encrpyt and decrypt values. The vault password and decrypted values are in the system memory, and could theoretically be written to disk if memory swapping is enabled.
 
 Authors
 -------
 
-Quipucords was originally written by Chris Hambridge <chambrid@redhat.com>, Noah Lavine <nlavine@redhat.com>, and Kevan Holdaway<kholdawa@redhat.com>.
+Quipucords was originally written by `Chris Hambridge <chambrid@redhat.com>`_, `Noah Lavine <nlavine@redhat.com>`_, and `Kevan Holdaway <kholdawa@redhat.com>`_.
```

Comment:
I would revert this as I don't know how the conversion to manpage format from RST will support it.

Reply:
I changed the format back. I thought that the contents of <> would not render in the view of the file in GitHub either. But they do in Rho. So I will check again when the new version of this file is merged.